dts/bindings/audio/nordic,nrf-pdm.yaml

```diff
@@ -5,7 +5,7 @@ description: Nordic PDM (Pulse Density Modulation interface)
 
 compatible: "nordic,nrf-pdm"
 
-include: [base.yaml, pinctrl-device.yaml]
+include: [base.yaml, pinctrl-device.yaml, nordic,easydma-device.yaml]
```

Comment:
If this commit is to add such inclusions for _all_ peripherals that use EasyDMA, as its message claims, at least the following are missing: AAR, CCM, ECB, I2S, QSPI, RADIO, SAADC, USBD.

Reply:
Together with @mbolivar-nordic, we decided to add this property only to peripherals that utilize it in shims so currently it is supported only in UARTE and SPIM, other peripherals will be aligned in the future